app/Models/LegiScan/Bill/History.php

```diff
@@ -24,15 +30,19 @@ class BillHistory extends Model
         'history_action',
     ];
 
-    #region Properties
+    protected $casts = [
+        'history_date' => 'datetime:Y-m-d',
```

Comment:
... That's literally contradictory.

"date TIME" <-> "Y-m-d"  ???

Reply:
I wouldn't do this in general. Either just do a date column, or format the datetime how you want it later. This is... yuck.